app/build.gradle

```diff
@@ -18,12 +18,20 @@ android {
 
 dependencies {
 
-    apt 'com.raizlabs.android:DBFlow-Compiler:1.4.5'
-    mod 'com.raizlabs.android:{DBFlow, DBFlow-Core}:1.4.5'
+    apt 'com.raizlabs.android:DBFlow-Compiler:2.0.0'
+    compile "com.raizlabs.android:DBFlow-Core:2.0.0"
```

Comment:
#req not sure why this changed. Just use `mod 'com.raizlabs.android:{DBFlow, DBFlow-Core}:2.0.0'